junebug/plugins/nginx/vhost.template

```diff
@@ -0,0 +1,4 @@
+server {
+  server_name %(server_name)s;
+  include %(includes)s;
+}
```

Comment:
There's usually a `listen 80;` here too, but I think if you leave it out it's the implicit default.